np.Templating/plugin.json

```diff
@@ -98,6 +98,22 @@
       "alias": [],
       "description": "np.Templating About...",
       "jsFunction": "templateAbout"
+    },
+    {
+      "name": "np:tft",
+      "alias": [
+        "Template file by title"
+      ],
+      "description": "Run a template file by title/preset",
+      "jsFunction": "templateFileByTitle"
+    },
+    {
+      "name": "templateRunner",
```

Comment:
This goes against the naming conventions used for all commands.  You have it right in the previous command.

e.g.

"name: "np:tr"

Reply:
Thx for the review @mikeerickson. Great comments. w/r/t the commands, I did this on purpose though. When we are finished testing, I plan to hide these commands from the UX, so no user will ever see them as `np:*` commands. But I needed a reasonable command name for users to use in the x-callback: `noteplan://x-callback-url/runPlugin?pluginID=np.Templating&command=templateRunner`, and I'm a bit afraid of colons in the xcallback context. Does that make sense or do you still want me to change it? `noteplan://x-callback-url/runPlugin?pluginID=np.Templating&command=np:tr`

Reply:
Ah gotcha, then nothing to see here :-)